app/celery/tasks.py

```diff
@@ -78,12 +86,14 @@ def send_sms(service_id, notification_id, encrypted_notification, created_at):
             created_at=created_at,
             sent_at=sent_at,
             sent_by=client.get_name()
-
         )
         dao_create_notification(notification_db_object)
 
         try:
-            client.send_sms(notification['to'], template.content)
+            client.send_sms(
+                notification['to'],
+                template.replaced
+            )
         except FiretextClientException as e:
             current_app.logger.debug(e)
```

Comment:
That line was added by here: https://github.com/alphagov/notifications-api/commit/a2341be0e2c200c1e8dc541b0608876f6b09739c#diff-201d2d704e6c87ac11b55391f2da412fR28

@minglis is there a good reason for it not to be an exception?